### PR TITLE
Branding

### PR DESCRIPTION
**Pull request by joaander**
## Description

Add Glotzer Group and University of Michigan branding to the documentation page footer. This template is suitable for use with other Glotzer Group projects.

## Preview

[Documentation preview](https://fresnel.readthedocs.io/en/branding/)

## Change log

n/a

## Checklist:
- [x] I have reviewed the [**Contributor Guidelines**](https://github.com/glotzerlab/fresnel/blob/master/CONTRIBUTING.md).
- [x] I agree with the terms of the [**Fresnel Contributor Agreement**](https://github.com/glotzerlab/fresnel/blob/master/ContributorAgreement.md).
- [x] My name is on the [list of authors](https://github.com/glotzerlab/fresnel/blob/master/doc/credits.rst).

### CODE REVIEW

File: doc/_templates/footer.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  {% endif %}
+
+  <hr/>
+
+  <div role="contentinfo">
+    <a href="http://umich.edu">
+    <img src="{{ pathto('_static/umich-block-M.svg', 1) }}" alt="University of Michigan logo" style="width: 100px; float: right;" />
+    </a>
+
+    <p>{{ project }} development is led by the <a href="https://glotzerlab.engin.umich.edu">Glotzer Group</a> at the <a href="https://umich.edu">University of Michigan</a>
```
Comment (bdice) [inline doc/_templates/footer.html]: The order will probably be flipped to `Development of {{ project }}` for freud and signac. As a stylistic rule, we avoid starting sentences with the (lowercase) package name.

Comment (bdice) [reply doc/_templates/footer.html]: Do we wish to be known as the Glotzer Group or the Glotzer Lab? Our naming is pretty inconsistent between the URL and page titles. I think @kjcoulter gave a guideline about this once but I can't recall it.

Comment (bdice) [reply doc/_templates/footer.html]: I would add a period at the end of this sentence.

**Comment by kjcoulter** [comment]
Thanks for chance to review -- looks good.   @bdice -- We are always "Glotzer Group" in text, documentation and branding.  We do use glotzerlab internally for a number of resources, but not as a label for collective work.

**Comment by joaander** [comment]
Thanks!

**Pull request merged**